src/vxingest/prepbufr_to_cb/vx_ingest_manager.py

```diff
@@ -0,0 +1,488 @@
+                                    for variable in sorted(
+                                        list(pb_interpolated_120.keys())
+                                    ):
+                                        self.debug_station_file.write(
+                                            f"level:{level} {variable}: {pb_interpolated_120[variable].get(level, None) if pb_interpolated_120[variable] is not None else None}\n"
+                                        )
+
+                                # interpolated data
+                                # WIND report type 220 interpolated data
+                                self.debug_station_file.write(
+                                    f"\nWIND report type 220 interpolated data station:{station} level:{level}\n"
+                                )
+                                if level in pb_interpolated_220["pressure"]:
+                                    for variable in sorted(
+                                        list(pb_interpolated_220.keys())
+                                    ):
+                                        self.debug_station_file.write(
+                                            f"level:{level} {variable}: {builder.interpolated_data[station][220]["data"][variable].get(level, None)}\n"
+                                        )
+
+                                # write station data
+                                self.debug_station_file.write(
+                                    f"\ndocument_map data for station:{station} level:{level}\n"
+                                )
+                                r = re.compile(f"DD:V01:RAOB:obs:prepbufr:{level}:.*")
+                                key = list(filter(r.match, document_map.keys()))[0]
+                                pb_final = document_map[key]["data"][station]
+                                self.debug_station_file.write("\n")
+                                self.debug_station_file.write(
+                                    json.dumps(
+                                        pb_final,
+                                        indent=2,
+                                    )
+                                )
+                                date = document_map[key]["fcstValidISO"].split("T")[0]
+                                stmnt_mysql = f'select wmoid,press,z,t,dp,rh,wd,ws from ruc_ua_pb.RAOB where date = "{date}"  and  press = {level} and wmoid = "{station}";'
+                                _mysql_db = mysql.connector.connect(
+                                    host=self.load_spec["_mysql_host"],
+                                    user=self.load_spec["_mysql_user"],
+                                    password=self.load_spec["_mysql_pwd"],
+                                )
+                                my_cursor = _mysql_db.cursor()
+                                my_cursor.execute(stmnt_mysql)
+                                my_result_final = my_cursor.fetchall()
```

Comment:
I missed this on my initial walkthrough - do we need to connect to a MySQL database? I thought VxIngest was Couchbase-only so that we could move away from our old MySQL DB.

And - if we do need that data, is there any other possible way to get it? I'd really like to avoid making the new ingest dependent on the old ingest.

Reply:
only for generating these reports (for analyzing the differences in the legacy ingest and the new ingest). It would probably be impossible without the reports. Once we have the analysis behind us we can remove or disable this stuff.